core/src/main/java/com/opensymphony/xwork2/config/impl/DefaultConfiguration.java

```diff
@@ -583,11 +590,10 @@ public ActionConfig getActionConfig(String namespace, String name) {
             }
 
             // fail over to empty namespace
-            if (config == null && StringUtils.isNotBlank(namespace)) {
+            if (config == null && StringUtils.isNotBlank(namespace) && ("/".equals(namespace) || fallbackToEmptyNamespace)) {
```

Comment:
I would extract this logic into a private method to name it, something like
```java
if (config == null && shouldFallbackToEmptyNamespaces(namespace)) {
    ...
```

Reply:
Done. Thanks